script.js

```diff
@@ -1,46 +1,61 @@
+            child.className = child.className === 'rotate' ?
+                'arrow' :
+                'rotate'
         }
     }
 }
-var pText = [];
-pText[0] = "<em>Java</em> Players attempt to avoid pathfinding enemies in a pseudorandomly generating world, or play in a custom-built world.";
-pText[1] = "<em>Java</em> Users interact with a dynamic map of the city of Berkeley and find the shortest path between point A and point B.";
-pText[2] = "<em>Python</em> In a parody of <em>Plants Vs. Zombies</em>, players attempt to prevent the Bees from reaching the Ant Queen with a number of tactical Ants.";
-pText[3] = "<em>Java</em> Users can enter a sequence of characters and get a nice melody outputted using the Karplus-Strong algorithm to convert it to \"plucking\".";
-pText[4] = "<em>Python</em> This was a group project to see whether a training an algorithm on short-term trends and financial reports could outperform the market."
-var pTitles = [];
-pTitles[0] = "Build Your Own World";
-pTitles[1] = "BearMaps";
-pTitles[2] = "Ants vs. Bees";
-pTitles[3] = "Digital Rockstar";
-pTitles[4] = "Beating the Market";
+
+const pText = [
+    "<em>Java</em> Players attempt to avoid pathfinding enemies in a pseudorandomly generating world, or play in a custom-built world.",
+    "<em>Java</em> Users interact with a dynamic map of the city of Berkeley and find the shortest path between point A and point B.",
+    "<em>Python</em> In a parody of <em>Plants Vs. Zombies</em>, players attempt to prevent the Bees from reaching the Ant Queen with a number of tactical Ants.",
+    "<em>Java</em> Users can enter a sequence of characters and get a nice melody outputted using the Karplus-Strong algorithm to convert it to \"plucking\".",
+    "<em>Python</em> This was a group project to see whether a training an algorithm on short-term trends and financial reports could outperform the market."
+]
+
+const pTitles = [
+    "Build Your Own World",
+    "BearMaps",
+    "Ants vs. Bees",
+    "Digital Rockstar",
+    "Beating the Market"
+]
+
+/*
+Alternative with a large object literal:
+  const projects = {
+      byow: {
+          title: "Build your Own World",
+          text: "<em>Java</em Players..."
+      },
+      map: { ... },
+      // ...
+  }
+
+  // ...then no index is required.
+  function alternate(proj) {
+      const part = // ...
+      const text = // ...
+      const {title, text} = projects[proj]
+      text.innerHTML = text.innerHTML === title ? text : title
+  }
+*/
 
 function alternate(proj, index) {
-    let part = document.getElementsByClassName(proj.toString())[0];
-    let text = part.getElementsByTagName("P")[0];
-    if (text.innerHTML == pTitles[index]) {
-        text.innerHTML = pText[index];
-    } else {
-        text.innerHTML = pTitles[index];
-    }
+    const text = document.querySelector(`#projects .${proj} p`)
+    text.innerHTML = text.innerHTML == pTitles[index] ?
+        pText[index] :
+        pTitles[index]
```

Comment:
[`document.querySelector`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector) lets you grab a DOM element using a CSS selector. Here, I reference a `<p>` which is a child of an element with class `proj` which is a child of the element with ID `projects`. In this case, there exists exactly one element that satisfies those criteria for any given `proj`, but in other situations you can use `document.querySelectorAll()` to get all elements which match the given selector.